client/src/document/organisms/sidebar/index.scss

```diff
@@ -16,15 +16,15 @@
 
   em {
     background-color: var(--background-toc-active);
+    border-bottom-right-radius: 0.25rem;
     border-left: 2px solid var(--category-color);
+    border-top-right-radius: 0.25rem;
     display: inline-block;
     font-style: normal;
     font-variation-settings: normal;
     font-weight: 600;
     hyphens: auto;
-    padding: 0.25rem;
-    padding-left: 0.5rem;
-    width: 100%;
```

Comment:
Apply the same changes to the curriculum sidebar: https://developer.allizom.org/en-US/curriculum/core/semantic-html/#2.6_media

Reply:
Fixed in 9f073f454a7b507e3c93b542e118b994af2bef8a.